proglearn/voters.py

```diff
@@ -14,12 +14,15 @@
 
 from sklearn.utils.multiclass import check_classification_targets
 
+from sklearn.base import BaseEstimator, ClassifierMixin
+
 from .base import BaseVoter
 
 
-class TreeClassificationVoter(BaseVoter):
+class TreeClassificationVoter(BaseVoter, BaseEstimator, ClassifierMixin):
```

Comment:
You don't need to inherit from BaseEstimator. By nature of inheriting from BaseVoter, it inherits from BaseEstimator.